package.json

```diff
@@ -25,24 +25,24 @@
   },
   "dependencies": {
     "@csstools/sass-import-resolve": "^1.0.0",
-    "postcss": "^7.0.14",
-    "sass": "^1.16.1",
+    "postcss": "^8.2.4",
```

Comment:
Move to peerDependencies - https://evilmartians.com/chronicles/postcss-8-plugin-migration

Reply:
I have moved postcss to the peerDependency.